docs/api/ethereum.md

```diff
@@ -1009,6 +1009,32 @@ const accountId = '0xcc567f78e8821fb8d19f7e6240f44553ce3dbfce';
 const abilities = await ledger.getAbilities(accountId);
 ```
 
+### getAccountAssetIdAt(accountId, index)
+
+An `asynchronous` class instance `function` which returns the asset id at specified `index` for desired `accountId`.
+
+**Arguments:**
+
+| Argument | Description
+|-|-
+| accountId | [required] A `string` representing the Ethereum account address.
+| index | [required] A `number` representing the asset index.
+
+**Result:**
+
+A `number` representing the asset id.
+
+**Example:**
+
+```ts
+// arbitrary data
+const accountId = '0xcc567f78e8821fb8d19f7e6240f44553ce3dbfce';
+const index = '100';
```

Comment:
Maybe set this at 0 or 1 since it is a more likely position.

Reply:
And add a comment that this account owns 3 assets so indexes 0,1,2 are available or something like this.